src/components/WithValidation/WithValidation.test.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+
```

Comment:
Remove the empty line, please
See code style for imports sequence here - https://wiki.folio.org/pages/resumedraft.action?draftId=14458784&draftShareId=483fa2fb-615e-4002-a916-c976bd5fb8c6&